### PR TITLE
Update messages in genesis #958

### LINKED ISSUE #958: Remove closemssg from genesis

### PR DESCRIPTION
**Pull request by boostmultifruit**
Resolves #958.

### CODE REVIEW

**Comment by boostmultifruit** [comment]
# :warning: 
Merge in same time with https://github.com/GolosChain/golos.contracts/pull/783

**Comment by boostmultifruit** [comment]
@zxcat  you can merge

File: programs/create-genesis/genesis_create.cpp [hunk 1]
```diff
@@ -1003,33 +1003,6 @@ struct genesis_create::genesis_create_impl final {
...
-        transaction tx{};
-        tx.actions.emplace_back(action{
-            {{_info.golos.names.posting, config::active_name}},
-            _info.golos.names.posting,
-            N(closemssg),
-            {}
-        });
-        const auto expiration_time = fc::hours(_info.golos.posts_trx.expiration_hours);
-        const auto delay = !_info.golos.posts_trx.initial_from ? microseconds{0} :
-            time_point_sec{_conf.initial_timestamp} - *_info.golos.posts_trx.initial_from;
```
Comment (s-medvedev) [inline programs/create-genesis/genesis_create.cpp]: Please remove from `genesis-info` parameters for closemssg generated transactions (_info.golos.posts_trx)

**Comment by zxcat** [comment]
@boostmultifruit, almost sure it's broken now. you should update `stored_contract_tables `: https://github.com/GolosChain/cyberway/blob/7c96fdd41cc1637b5156ece87eb2b7fa012a84c1/programs/create-genesis/serializer.hpp#L24-L28 if adding/removing sections

**Pull request merged**